libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts

```diff
@@ -16,9 +16,24 @@
  */
 import { WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
 
-export function enumToDropdownOptions<T extends object>(enumObj: T) {
-  return Object.keys(enumObj).map((key) => ({
-    displayValue: key,
-    value: Object.values(enumObj)[Object.keys(enumObj).indexOf(key)],
-  })) as WattDropdownOptions;
+type Sort = 'asc' | 'desc';
+
+export function DhEnumToWattDropdownOptions<T extends object>(
+  enumObj: T,
+  exclude?: string[],
```

Comment:
nitpick: filter the unnecessary options out before calling the function. There's no need to create options just to be discarded in the next step.

Note: this comment is more of a personal preference (thus marked as "nitpick"). The current version is fine too. Ignore the comment if you prefer the current version.

Reply:
But you dont have a list before this function. You only have an enum. So it needs to be made into a list before the filter can be applied :-)

Reply:
Oh, yeah, that's true. Let's keep it then :)